lib/fog/compute/google/models/instance_group_managers.rb

```diff
@@ -0,0 +1,22 @@
+module Fog
+  module Compute
+    class Google
+      class InstanceGroupManagers < Fog::Collection
+        model Fog::Compute::Google::InstanceGroupManager
+
+        def all(filters = {})
+          if filters[:zone]
+            data = Array(service.list_aggregated_instance_group_managers(filters[:zone]))
```

Comment:
Why do you need to cast this as an `Array`? What does it return normally?

Reply:
Fixed.